src/press/models.py

```diff
@@ -344,6 +346,21 @@ def navigation_items(self):
     def code(self):
         return 'press'
 
+    @property
+    def active_journals(self):
+        from journal import models as journal_models
+        # imported here to avoid circular imports
+        return journal_models.Journal.objects.filter(
+            hide_from_press=False,
+        )
+
+    @property
+    def published_articles(self):
+        return submission_models.Article.objects.filter(
+            stage=submission_models.STAGE_PUBLISHED,
+            date_published__lte=timezone.now(),
+        )
+
```

Comment:
as above, best to get the model with https://docs.djangoproject.com/en/3.2/_modules/django/apps/config/#AppConfig.get_model to avoid inevitable future circular imports

Reply:
For this one we still need submission_models.STAGE_PUBLISHED

Reply:
> For this one we still need submission_models.STAGE_PUBLISHED

Yep, one day we will also move these to a consts.py module to avoid circularity